Client/Frontend/Home/DefaultBrowserCard.swift

```diff
@@ -65,6 +66,7 @@ class DefaultBrowserCard: UICollectionViewCell {
         background.addSubview(closeButton)
         
         labelView.axis = .vertical
+        labelView.alignment = .leading
```

Comment:
I sneaked in this little fix for default browser card 😊 please be forgiving 🙏

Reply:
Sneaky fixes is better than no fixes 👍